### PR TITLE
Hide hearts remove x add copy delete in edit mode

### LINKED ISSUE #180: Hide the hearts in edit mode + remove x for edit mode + add copy/delete/done buttons
backend/frontend

### PR DESCRIPTION
**Pull request by mtherber**
### After creating your PR, please complete this checklist!

- [x] Go into cse110-fa21-group22/source/\_\_test\_\_/end-to-end.test.js
- [x] Change `const prNum = 0;` -> `const prNum = {your pr number};` _This should be on line 18_
- [x] Now push your change to the branch and it will rerun the checks

You are ready to finish the rest of the template.

---

### What were your changes

- Removed delete 'x' from recipe cards
- Hid hearts on recipe cards in edit mode
- Added delete in edit mode
- Added copy in edit mode
- Polished empty UserLists (they now show text and are smaller)
- Added a keybind (esc) for deselecting, exiting edit mode, or editing copy mode (in that order)
- Added text at the top of the favorites page that tells a user to select a recipe when they haven't selected anything
- Formatted favorites page edit mode for mobile

---

### What specifically should be checked

- Entering edit mode and deleting recipes
- Entering edit mode, then copy mode, then copying recipes
- Copying recipes to a list where they were previously deleted
- Deleting everything in a list
- Deleting from multiple lists at the same time
- Copying to a list where some or all of the selected items exist
- Esc keybind should deselect, exit edit mode, or exit copy mode (in that order of precedence)
- Favorites page should look good on mobile
- Cancel button should say "Save" after a change has been made
- Text should appear at top middle of favorites page when nothing is selected and in edit or copy mode
- Hearts are no longer visible in edit or copy mode
- Delete 'x' icons are no longer on recipe cards

### CODE REVIEW

**Comment by mtherber** [comment]
I ran prettier and merged but prettier is still failing. I'm not sure what to do about it.

File: styles/recipe.css [hunk 1]
```diff
@@ -29,9 +29,9 @@ h3 {
   margin-bottom: 12px;
   cursor: pointer;
   font-size: 22px;
-  -webkit-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
+  user-select: none;
```
Comment (HairforceNiko) [inline styles/recipe.css:32]: I think you don't need to delete these user-select

File: styles/recipeCard.css [hunk 2]
```diff
@@ -17,9 +17,9 @@
   margin-bottom: 12px;
   cursor: pointer;
   font-size: 22px;
-  -webkit-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
+  user-select: none;
```
Comment (HairforceNiko) [inline styles/recipeCard.css:20]: Same as above

**Comment by HairforceNiko** [comment]
I think the reason why prettier fail is the line:  import "reset.css" in the css files.
What you need to do is change "" to ''

**Review by HairforceNiko — APPROVED** [review]

**Pull request merged**